bears/general/ShebangBear.py

```diff
@@ -0,0 +1,33 @@
+import os
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
+
+    def run(self, filename, file):
+        """
+        Ensure that the file uses the generic Shebang operator.
+        """
+        if len(file) == 0 or '#!/usr/bin/env' in file[0]:
+            return
+
+        diff = Diff(file)
+        message = None
+
+        if '#!' in file[0]:
+            binary = os.path.basename(file[0]).strip('\n')
+            new_line = '#!/usr/bin/env {}'.format(binary)
+            diff.modify_line(1, new_line)
+            message = 'Use the {}.'.format(repr(new_line))
```

Comment:
"Use `/usr/bin/env`" and then a more info "`/usr/bin/env` is preferable to be used because blah blah blah ..."